package.json

```diff
@@ -21,7 +21,8 @@
     "record-build-info": "node ./bin/record-build-info",
     "lint": "eslint . --cache --max-warnings 0",
     "typecheck": "tsc",
-    "test": "jest --runInBand --detectOpenHandles --collectCoverage=true --testPathPattern=/server/.*",
+    "test": "jest --collectCoverage=true --testPathPattern=/server/.*",
+    "test:ci": "jest --maxWorkers=2 --collectCoverage=true --testPathPattern=/server/.*",
```

Comment:
circle reports wrong machine details and jest locks if unbounded. Two workers seems to work ok on other projects.